src/Microsoft.TestPlatform.ObjectModel/Client/Events/DiscoveryCompleteEventArgs.cs

```diff
@@ -50,19 +50,25 @@ public DiscoveryCompleteEventArgs(long totalTests, bool isAborted)
     /// Gets the list of sources which were fully discovered.
     /// </summary>
     [DataMember]
-    public IList<string> FullyDiscoveredSources { get; set; } = new List<string>();
+    public IList<DiscoveredSource> FullyDiscoveredSources { get; set; } = new List<DiscoveredSource>();
 
     /// <summary>
     /// Gets the list of sources which were partially discovered (started discover tests, but then discovery aborted).
     /// </summary>
     [DataMember]
-    public IList<string> PartiallyDiscoveredSources { get; set; } = new List<string>();
+    public IList<DiscoveredSource> PartiallyDiscoveredSources { get; set; } = new List<DiscoveredSource>();
+
+    /// <summary>
+    ///  Gets a list of sources that were skipped during discovery.
```

Comment:
Other doc comments are broken too but this should be `Gets or sets`.

Reply:
done for all